packages/shared-lib/src/input-gateway/deviceTriggerPreviews.ts

```diff
@@ -61,4 +61,6 @@ export type PreviewWrappedAdLib = Omit<IWrappedAdLibBase, '_id'> & {
 		  }
 		| undefined
 	styleClassNames: string | undefined
+	isCurrent: boolean | undefined
```

Comment:
I'm not sure about calling this state `isCurrent`. My reading is that "current" is a state of the Part, whereas this doesn't just mean "AdLib is current", it actually means "AdLib is playing", since this will be reactive to actual playout and any limiations of the corresponding Piece's `enable` property. Maybe an `isOnAir` would make the meaning more clear?